dapp/src/pages/DashboardPage/TransactionHistory/TransactionTable.tsx

```diff
@@ -1,96 +1,115 @@
+                    {displayBlockTimestamp(getActivityTimestamp(activity))}
                   </TextSm>
 
-                  <HStack flexBasis="60%">
+                  <HStack flexBasis="55%">
                     <TextSm
                       color="grey.700"
                       flex={1}
                       fontWeight="semibold"
                       textTransform="capitalize"
                     >
-                      {type}
+                      {activity.type}
                     </TextSm>
 
                     <Box flex={1}>
                       <CurrencyBalance
                         color="grey.700"
                         size="sm"
                         fontWeight="bold"
-                        amount={amount}
+                        amount={activity.amount}
                         currency="bitcoin"
+                        withDots
                       />
                     </Box>
                   </HStack>
-                  {txHash && (
+                  {activity.txHash ? (
                     <BlockExplorerLink
-                      id={txHash}
+                      id={activity.txHash}
                       chain="bitcoin"
                       type="transaction"
-                      variant="ghost"
-                      color="grey.300"
-                      _groupHover={{ color: "brand.400" }}
-                      pl={6}
-                      pr={4}
-                      py={5}
-                      mx={-4}
-                      my={-5}
+                      color="grey.600"
+                      _groupHover={{
+                        color: "brand.400",
+                        textDecoration: "none",
+                      }}
+                      minW={BLOCK_EXPLORER_CELL_MIN_WIDTH}
                     >
-                      <IconArrowUpRight size={16} />
-                      <VisuallyHidden>View transaction details</VisuallyHidden>
+                      <HStack spacing={1}>
+                        <TextSm>Details</TextSm>
+                        <Icon
+                          as={IconArrowUpRight}
+                          color="brand.400"
+                          boxSize={4}
+                        />
+                      </HStack>
                     </BlockExplorerLink>
+                  ) : (
+                    <Box minW={BLOCK_EXPLORER_CELL_MIN_WIDTH} />
                   )}
-                </CardBody>
-              </Card>
-            ),
-          )
+                </HStack>
+                <EstimatedDuration activity={activity} />
+              </CardBody>
+            </Card>
+          ))
         }
       </PaginationPage>
 
-      <HStack
+      <PaginationFooter
         spacing={2}
-        mx={-5}
-        mt={-6}
-        mb={-10}
-        p={5}
-        pt={6}
-        bgGradient="linear(to-b, transparent, gold.200 20%)"
-        zIndex={2}
+        // TODO: Temporary solution - Animation should be fixed in such a way
```

Comment:
Could you give an example of the issue or elaborate some more? I don't understand the issue to be honest.

Reply:
You can find more details in https://github.com/thesis/acre/issues/705. There is an example video of issue.